codegen/lib/graphql_java_gen/templates/APISchema.java.erb

```diff
@@ -265,6 +265,7 @@ public class <%= schema_name %> {
                 <% end %>
                 <% type.optional_input_fields.each do |field| %>
                   private <%= java_input_type(field.type) %> <%= escape_reserved_word(field.camelize_name) %>;
+                  private boolean <%= escape_reserved_word(field.name) %>Seen = false;
```

Comment:
No java reserved words end in `Seen`, so you don't need `escape_reserved_word`.

Also, I think you should use `field.camelize_name` to follow the java naming convention if the schema uses underscore separated words.  E.g. if the field in the schema is `two_words` then the variable would be `two_wordsSeen`.